### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by davidterranova** [erikdahlstrand/shrine-rails-example#2]
It could be a good idea to provide the tests for this shrine example.

regards,

**Comment by janko-m** [comment]
I agree, that is a good idea, it would give more value to the implementation, I will get working on those.

**Comment by janko-m** [comment]
I decided that adding and maintaining tests for the example app isn't worthwhile. Mostly because I find writing end-to-end tests slighly PITA. Also, since I would have to communicate directly to S3, it would be difficult for me to set the timeouts which won't be too short for slower connections. I described in the README exactly how the app works, that should hopefully be enough.

**Comment by olingern** [comment]
@janko-m just wanted to comment that having some basic `specs` would be helpful

**Comment by janko-m** [comment]
The official demo app has an [acceptance test](https://github.com/shrinerb/shrine/blob/b1e5d8362a8204e287d1609b580ccb90f8f009d6/demo/test/acceptance_test.rb), would something like that suffice? (just that we port if over here)

**Comment by olingern** [comment]
> would something like that suffice?

Yes! This is great. Thanks for pointing me toward that
